### PR TITLE
docs: Update system table schema in docs/sql.md

### PR DESCRIPTION
**Pull request by capkurmagati**
Describe your proposed changes here.
- Column `count` is renamed to `row_count`  via #1937
- System tables splits into smaller ones  #2061
- This PR updates the sql to reflect those changes.

- [x] I've read the contributing section of the project [README](https://github.com/influxdata/influxdb_iox/blob/main/README.md).
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed).

### CODE REVIEW

File: docs/sql.md [hunk 1]
```diff
@@ -198,8 +198,8 @@ LIMIT 20;
 ### Total row count by table
 
 ```sql
-SELECT database_name, table_name, max(count) as total_rows
-FROM columns
+SELECT database_name, table_name, max(row_count) as total_rows
```
Comment (capkurmagati) [inline docs/sql.md:201]: This query does not actually return the `Total row count by table`.
Instead, it returns the row count of the largest partition by table.
I think we might need something like:
```
SELECT database_name, table_name, sum(total_rows) as total_rows
FROM (
  SELECT database_name, table_name, max(row_count) as total_rows
  FROM chunk_columns
  GROUP BY database_name, partition_key, table_name
)
GROUP BY database_name, table_name
ORDER BY total_rows DESC;
```

Comment (alamb) [reply docs/sql.md:201]: yes I agree -- nice catch @capkurmagati 👍

File: docs/sql.md [hunk 2]
```diff
@@ -208,9 +208,9 @@ LIMIT 20;
 ### Total row count by partition and table
```
Comment (alamb) [inline docs/sql.md:208]: Given the query has been changed, I think we should also change the section title

```suggestion
### Total row count by table
```

**Review by alamb — COMMENTED** [review]
Thank you @capkurmagati . This is looking good

File: docs/sql.md (hunk 2 above)
Comment (capkurmagati) [reply docs/sql.md:208]: Ah, I accidentally removed the `partition_key`. Let me put it back.

File: docs/sql.md (hunk 1 above)
Comment (capkurmagati) [reply docs/sql.md:201]: Thanks! I will update the pr and rebase against main branch.

**Comment by capkurmagati** [comment]
@alamb PR updated. PTAL.

**Review by alamb — APPROVED** [review]
Looks great -- thank you @capkurmagati !

**Comment by alamb** [comment]
For some reason circle CI checks don't appear to be running for this PR.

I think this repo is setup correctly according to https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories (has "build forked pull requests"). However that page also says

> Note: If a user submits a pull request to your repository from a fork, but no pipeline is triggered, then the user most likely is following a project fork on their personal account rather than the project itself of CircleCi, causing the jobs to trigger under the user’s personal account and not the organization account. To resolve this issue, have the user unfollow their fork of the project on CircleCI and instead follow the source project. This will trigger their jobs to run under the organization when they submit pull requests.

Does this make sense to you @capkurmagati ? If not, I will just pull your changes into another PR and we'll get it merged that way but I figured I would ask if you had the time

**Comment by capkurmagati** [comment]
@alamb Thanks for the information.
I'm not sure if there is any step I've missed, but I couldn't find where the source project is.
I can only choose my org(the forked project is in there and I'm not following it). 
image
I wonder if my last PR #2290 triggered the CI.

**Comment by alamb** [comment]
Thanks @capkurmagati  --

> I wonder if my last PR #2290 triggered the CI.

It did indeed -- I wonder if it was related to when I rebased the PR. Let me add some minimal commit to this one and see if I can trigger the CI

**Comment by alamb** [comment]
Ok, the checks are running 🎉  I expect this PR to be merged imminently 🤞

**Comment by alamb** [comment]
🎉

**Comment by capkurmagati** [comment]
@alamb Thanks for the help. Much appreciated!

**Comment by alamb** [comment]
Thank you @capkurmagati  for the contribution

**Pull request merged**